src/test-harness/SparkEthereumTests.sol

```diff
@@ -625,16 +626,19 @@ abstract contract SparkEthereumTests is SparklendTests {
     {
         IMorphoChainlinkOracleV2 _oracle = IMorphoChainlinkOracleV2(oracle);
 
+        uint256 loanTokenDecimals = IERC20(loanToken).decimals();
+        uint256 ptDecimals        = IERC20(pt).decimals();
+
         assertEq(address(_oracle.BASE_FEED_2()),          address(0));
         assertEq(address(_oracle.BASE_VAULT()),           address(0));
         assertEq(_oracle.BASE_VAULT_CONVERSION_SAMPLE(),  1);
         assertEq(address(_oracle.QUOTE_FEED_1()),         address(0));
         assertEq(address(_oracle.QUOTE_FEED_2()),         address(0));
         assertEq(address(_oracle.QUOTE_VAULT()),          address(0));
         assertEq(_oracle.QUOTE_VAULT_CONVERSION_SAMPLE(), 1);
-        assertEq(_oracle.SCALE_FACTOR(),                  1e18);
+        assertEq(_oracle.SCALE_FACTOR(),                  1e18 * 10 ** (loanTokenDecimals - ptDecimals));
```

Comment:
Cache this value to a variable: `assetConversion`. Also add a note that this assumes `loanTokenDecimals >= ptDecimals`. Maybe add a TODO to revisit this. It won't come up for a bit but it could if we start lending PTs against USDC